### PR TITLE
feature / default output type

### PR DESCRIPTION
**Pull request by c-milocanovich**

### CODE REVIEW

File: src/components/query-viewer/controller/query-viewer-controller.tsx [hunk 1]
```diff
@@ -443,7 +467,7 @@ export class QueryViewerController {
       this.queryViewerServiceResponsePivotTable.emit({
         MetaData: metadata,
         metadataXML: metadataXML,
-        Properties: queryViewerBaseProperties,
+        Properties: { ...queryViewerBaseProperties, outputType: this.realType },
```
Comment (ncamera) [inline src/components/query-viewer/controller/query-viewer-controller.tsx:470]: Is `queryViewerBaseProperties` always defined? Doing `{...null, outputType: this.realType }` will throw an exception.

File: src/components/query-viewer/controller/query-viewer-controller.tsx [hunk 2]
```diff
@@ -463,7 +487,7 @@ export class QueryViewerController {
           this.queryViewerServiceResponse.emit({
             MetaData: metadata,
             Data: data,
-            Properties: queryViewerBaseProperties,
+            Properties: { ...queryViewerBaseProperties, outputType: this.realType },
```
Comment (ncamera) [inline src/components/query-viewer/controller/query-viewer-controller.tsx:490]: Same comment as before.

**Review by ncamera — CHANGES_REQUESTED** [review]
I left a small adjustment. Thanks!

File: src/components/query-viewer/controller/query-viewer-controller.tsx (hunk 1 above)
Comment (c-milocanovich) [reply src/components/query-viewer/controller/query-viewer-controller.tsx]: I added a condition to prevent this issue

**Review by ncamera — APPROVED** [review]

**Pull request merged**